src/analysis/souffle/authorization_logic.dl

```diff
@@ -28,11 +28,13 @@
 
 // Some party asserts the claim that the label of an access path should be
 // downgraded by removing the referenced tag from the set.
-.decl saysDowngrades(speaker: Principal, p: AccessPath, t: Tag)
+// This relation needs to be in snake case because outputs from the 
+// authorization logic will be in snake case.
+.decl says_downgrades(speaker: Principal, p: AccessPath, t: Tag)
```

Comment:
If these are really needed, can you pull it off to a separate PR?

Reply:
There is a comment that tries to explain the style difference, but the comment could be easy to miss. The code generated by the authorization logic uses snake case, so either this line needs to change or the output from the auth logic needs to change. The file `authorization_logic.dl` is just for defining the interface between the two things, so I thought it would be OK to change the style on this one line here.

Reply:
I can do it in a separate PR as long as it's OK to do proper builds and testing in a separate PR, because this change is needed to get it to work.

Reply:
Not necessary, and as we talked about today, we don't need to get an automated test working to get this in.